euphoria-local/src/main/java/cz/seznam/euphoria/executor/local/LocalExecutor.java

```diff
@@ -477,7 +477,6 @@ public void setAccumulatorProvider(AccumulatorProvider.Factory factory) {
                 // end of the stream
                 writer.flush();
                 writer.commit();
-                writer.close();
```

Comment:
I'm not sure here it writer's life cycle doesn't require calling `close`, even after `commit`. That should be clarified and documented.